src/api/GraphQuery/queryCRUD.ts

```diff
@@ -102,3 +102,21 @@ export interface ConvertQueryResponse {
   query: string;
   dialect: GraphQueryDialect;
 }
+
+export interface GetQueryStatsParams extends IDataSourceParams {
+  query: StructuredGraphQuery;
+  dialect: GraphQueryDialect.QUERY_BUILDER;
```

Comment:
As dialect is always queryBuilder, so it is mandatory to always pass it as param from the request ?

Reply:
The idea of Mahmoud was to prepare the API to support other dialect. That's debatable as we could also have considered adding the field later.